From 2849a226806a0e70cd85cca3d09bcd29438007dd Mon Sep 17 00:00:00 2001
From: Sami Kerola <kerolasa@iki.fi>
Date: Wed, 18 Jul 2012 21:13:29 +0200
Subject: [PATCH 05/44] Remove bash backtick command substitution

The $(command) is more flexible as it allows nesting and easier to
read, than `command` notation.

Signed-off-by: Sami Kerola <kerolasa@iki.fi>
(cherry picked from commit 64acedea081a94352f9b98a808747cbd25e989d4)
---
 virt-what.in | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/virt-what.in b/virt-what.in
index 4f381ab..d50426a 100644
--- a/virt-what.in
+++ b/virt-what.in
@@ -43,7 +43,7 @@ function usage {
 
 # Handle the command line arguments, if any.
 
-TEMP=`getopt -o v --long help --long version --long test-root: -n 'virt-what' -- "$@"`
+TEMP=$(getopt -o v --long help --long version --long test-root: -n 'virt-what' -- "$@")
 if [ $? != 0 ]; then exit 1; fi
 eval set -- "$TEMP"
 
@@ -78,19 +78,19 @@ fi
 # Many fullvirt hypervisors give an indication through CPUID.  Use the
 # helper program to get this information.
 
-cpuid=`virt-what-cpuid-helper`
+cpuid=$(virt-what-cpuid-helper)
 
 # Check for various products in the BIOS information.
 # Note that dmidecode doesn't exist on non-PC architectures.  On these,
 # this will return an error which is ignored (error message redirected
 # into $dmi variable).
 
-dmi=`LANG=C dmidecode 2>&1`
+dmi=$(LANG=C dmidecode 2>&1)
 
 # Architecture.
 # Note for the purpose of testing, we only call uname with -p option.
 
-arch=`uname -p`
+arch=$(uname -p)
 
 # Check for VMware.
 # cpuid check added by Chetan Loke.
-- 
1.8.3.1

